### DISCUSSION TITLE
Add links "in the wild"

### DISCUSSION THREAD

**Discussion by coolapso** [jcubic/jquery.terminal#781] [Show and tell]
Hi, 

I'd like to add some of my stuff to the "in the wild" section and i would do the PR myself however i don't know exactly in which section they fit it. 

There's my page: https://cv.4s3ti.net

And the HUGO engine theme that I have built and currently powering the one above: https://github.com/4s3ti/terminalCV

Can you please add them or let me know where they fit in the best and ill make the PR?

**Comment by jcubic** [comment]
Added next to other TerminalCV your project is not the only one.

**Reply by coolapso** [reply]
Cool, thanks a lot
